okta-hosted-login/README.md

```diff
@@ -0,0 +1,58 @@
+# Okta-Hosted Login Page Example
+
+This example shows you how to implement Authorization Code flow using [Micronaut][Micronaut Framework] to login a user. The login is achieved through the [Authorization Code Flow][] where the user is redirected to the Okta-Hosted login page. After the user authenticates successfully, they are redirected back to the application, and a local cookie session is created.
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
+* An Okta Application, configured for Web mode. This is done from the Okta Developer Console, and you can find instructions [here][OIDC Web Application Setup Instructions].
+* In the above application, configure `http://localhost:8080/oauth/callback/okta` as `Login redirect URI` and
```

Comment:
I think this needs to be `/oauth2`, not just `/oauth`. At least that's what I used in [my JHipster + Micronaut blog post](https://developer.okta.com/blog/2020/08/17/micronaut-jhipster-heroku). Then again, Micronaut's [docs](https://guides.micronaut.io/micronaut-oauth2-okta/guide/index.html) have what you have.

Reply:
yeah, I was sticking to Micronaut doc.